docs/reference/actions.md

```diff
@@ -131,7 +131,7 @@ The list of supported modifier keys is shown below.
 
 Sometimes a controller needs to listen for events dispatched on the global `window` or `document` objects.
 
-You can append `@window` or `@document` to the event name (along with any filter modifier) in an action descriptor to install the event listener on `window` or `document`, respectively, as in the following example:
+You can append `@window` or `@document` to the event name (along with any filter modifer) in an action descriptor to install the event listener on `window` or `document`, respectively, as in the following example:
```

Comment:
I assume this wasn't changed on purpose?

Reply:
Yeah. That's an accident, not sure what happened there sorry.

Reply:
PR up with a fix for this https://github.com/hotwired/stimulus/pull/704

Reply:
Thank you! 🙏🏼